Sources/SwiftParser/Lexer/Cursor.swift

```diff
@@ -1172,11 +1177,7 @@ extension Lexer.Cursor {
           break
         }
       case UInt8(ascii: "#"):
-        guard start.isAtStartOfFile, self.advance(if: { $0 == "!" }) else {
-          break
-        }
-        self.advanceToEndOfLine()
-        continue
+        break
```

Comment:
Actually, now that I read this again, we can just add `UInt8(ascii: "#")` to the `Punctuation` section a few lines further below. It doesn’t need its own case anymore.